data/incipias/incipias first contact.txt

```diff
@@ -0,0 +1,399 @@
+			choice
+				`	"Greetings, Incipias. I am a human."`
+				`	"I thank you for your hospitality."`
+			`	Your translator begins rapidly flashing. The Incipias seem to respond to this - movement and colors begin to diffuse through the crowd like waves.`
+			`	The 'speaker' continues. "You are the first outer-being that ever communicated with us. Are you the only outer-being out there?"`
+			choice
+				`	"No, there are others too."`
+				`	"The galaxy is full of other beings."`
+			`	Another wave goes through the crowd. "So there are more. But, far-stranger, how can we help you?"`
+			choice
+				`	(Tell them about the stranded ship and how I want to help them.)`
+					goto stranded
+				`	"I want one of your ships."`
+			`	"We shall allow you to buy one, if that is all that it takes to make you a not-danger. Please excuse us. We need to talk about what this new situation-opportunity means for our species. I hope we can out-question you more another time."`
+				goto end
+			label stranded
+			`	Another wave goes through the crowd, but this time you notice an overflow in green tones.`
+			`	"So they are still alive. We shall give you access to our ships, so that you may enter the air-planet they have taken refuge in. Do note though that they cannot jump between system-places, not like your own ship. We hope to question you more on another future-time.`
+			label end
+			`	The five float away but the crowd continues staring at you, probably out of interest.`
+
+mission "Incipias: No Help For The Stranded"
+	landing
+	invisible
+	source
+		planet Turra
+	destination "Pon'tes"
+	to offer
+		or
+			has "Incipias: Help The Stranded 1: declined"
+		not "Incipias: Help The Stranded 1: offered"
+	on offer
+		conversation
+			`You identify the stranded ship on your scanners, but with no way to communicate with them you are unable to to help. You have no choice but to leave them to their doom on this stormy world.`
+
+mission "Incipias: Help The Stranded 2"
+	landing
+	passengers 5
+	source
+		planet Turra
+	destination "Pon'tes"
+	to offer
+		has "Incipias: Help The Stranded 1: done"
+	on offer
+		conversation
+			`Your ship dives into the clouds of the gas giant, entering an environment on a scale you never experienced before. Giant clouds tower like castles, sometimes opening to giant caverns. Whirlwinds brush through the atmosphere like pillars holding the sky. Even though the winds are brutal your ship glides through them as if they had no effect at all.`
+			`	You activate your scanners looking for the stranded ship. Soon a small blip appears on your screen, and you maneuver your ship to that point. The stranded ship looks even worse than the first time; the storms seem to have damaged some of the hull, and their shields are completely down - the shield generator must have gone offline when they ran out of fuel.`
+			`	You receive no answer to your hail.`
+			`	You carefully maneuver your ship closer, until you can dock with the Incipias ship.`
```

Comment:
Would it not show up in the outfitter?

Reply:
If you buy a ship that has one, true, and if you were to go into the outfitter, if it appears there, yes. However, there's no guarantee the player does this or explicitly notes it, so I think "it must be the case" is a bit over-assuming versus "it could be the case".

Since this ship has been damaged in many ways, there's no telling if they ran out of fuel before or after their shields gave out; perhaps the storm damaged the device before they ran out of fuel.

So, saying "could" here is a bit more open-ended, and still acknowledges that the shield generator could stop working as a result of fuel outage.